### PR TITLE
Module load issue

### LINKED ISSUE #763: Bug - Admin site pages for Explore and Delete are broken
# Describe the bug
The admin site deploys fine, launches, and offers the Ingest data page which works. However the Explore and Delete pages are broken and have no functionality.

# Expected behavior
All admin pages working fine.

# How does this bug make you feel?
Unhappy that the back-end cannot be administered. Also disappointed that check-ins are happening that allow breaking changes like this. Bummed that Microsoft does not have a QA process for public Github repos to confirm pull requests do not result in breaking changes.

---

# Debugging information

## Steps to reproduce
1. One-click deploy the accelerator
2. Successful deployment, no errors reported
3. Launch the admin web site
4. Ingest some files
5. Go to Explore or Delete pages in the admin site
6. Neither works, has a large pink error message that says:

ModuleNotFoundError: No module named 'backend'
Traceback:
File "/usr/local/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 584, in _run_script
    exec(code, module.__dict__)
File "/usr/local/src/myscripts/admin/pages/02_Explore_Data.py", line 7, in <module>
    from batch.utilities.search.IntegratedVectorizationSearchHandler import (
File "/usr/local/src/myscripts/admin/batch/utilities/search/IntegratedVectorizationSearchHandler.py", line 1, in <module>
    from backend.batch.utilities.search.SearchHandlerBase import SearchHandlerBase

## Screenshots
image

## Logs

If applicable, add logs to help the engineer debug the problem.

---

# Tasks

_To be filled in by the engineer picking up the issue_

- [ ] Task 1
- [ ] Task 2
- [ ] ...

### PR DESCRIPTION
**Pull request by komalg1**
## Purpose

* Fixes the Explore page load error
* Fixes #763 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by ross-p-smith — APPROVED** [review]

**Review by ross-p-smith — DISMISSED** [review]
Tests failing

**Comment by adamdougal** [comment]
Can you change the PR title too? Thanks!

File: code/backend/batch/utilities/search/IntegratedVectorizationSearchHandler.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from backend.batch.utilities.search.SearchHandlerBase import SearchHandlerBase
```
Comment (superhindupur) [inline code/backend/batch/utilities/search/IntegratedVectorizationSearchHandler.py:1]: How come this was building fine but only failing after deployment?

Is there any way this type of problem can be prevented in the future?

**Review by adamdougal — APPROVED** [review]

File: code/backend/batch/utilities/search/IntegratedVectorizationSearchHandler.py (hunk 1 above)
Comment (komalg1) [reply code/backend/batch/utilities/search/IntegratedVectorizationSearchHandler.py:1]: This needs to be looked into. Will raise an issue for the same

**Pull request merged**